alert-city/mobile
# Feat/cp 41 update ios icon

## TL;DR

Update iOS icon

## How Has This Been Tested?
1. install iOS app
2. check icon from launcher
3.

## Screenshots (if applicable)


## Files for Review
all change diffs

## Resolve
Resolve CP-41


## Additional Notes
